tests/conftest.py

```diff
@@ -2686,6 +2802,8 @@ def cloud_data_context_in_cloud_mode_with_datasource_pandas_engine(
         "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend.list_keys"
     ), mock.patch(
         "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend._set"
+    ), mock.patch(
```

Comment:
Now we need to also mock the new get request kicked off when we call `DatasourceStore.set()`